Libplanet.Unity/SwarmRunner.cs

```diff
@@ -37,6 +37,16 @@ public class SwarmRunner
             _cancellationTokenSource = new CancellationTokenSource();
         }
 
+        /// <summary>
+        /// EventHandler checked if IsPreloaded has changed.
+        /// </summary>
+        public delegate void EventHandler();
```

Comment:
As following microsofts naming convention, seems there're more option to naming this event handler.
https://docs.microsoft.com/en-us/dynamicsax-2012/developer/naming-conventions-delegates-and-event-handlers